### ISSUE TITLE
Colors plugin example not working

### ISSUE THREAD

**Issue by btecu** [Alex-D/Trumbowyg#304]
Following the example for `colors` plugin does not add the buttons:

```
$('#editor')
.trumbowyg({
  // You can only add one of foreColor/backColor
  btnsAdd: ['foreColor', 'backColor']
});
```

However, using `btns` instead works:

```
btns: [...,
  ['foreColor', 'backColor']
]
```

Using `btns` requires listing all other buttons.

**Comment by Alex-D** [comment]
Hi!

In fact, since v2 stable that option has changed, I need to update all examples and improve documentation.

Thanks for your feedback :)

**Comment by zevero** [comment]
oh please update documentation. Its not encouraging to read 

> WTF are btnGrp-* buttons?

Well I managed now to get on, but I can not change the colors as I did before using 

    $.trumbowyg.opts.colors = [];

**Comment by Alex-D** [comment]
Exemple fixed in 2.8.0 which will be released soon.
